shell/directives/strip-aria-label.js

```diff
@@ -0,0 +1,16 @@
+export default {
```

Comment:
shouldn't this be called `clean-aria-labels` and use the same process at clean-tooltips?

Reply:
we could call it `clean-aria-labels` but it's different from `clean-tooltips` for sure since tooltips may accept HTML, which is sanitized first, whereas the aria-labels one is for stripping the HTML tags (we don't want HTML on aria-labels), hence the name

Reply:
ah, got it now.

that should be covered in a comment or update the name to be something like `strip-html-aria-label`

Reply:
Updated the name of the file and added comment to file directive